src/main/java/com/champstart/recipeapp/recipe/model/RecipeModel.java

```diff
@@ -28,16 +28,17 @@
 public class RecipeModel {
     @Id
     @GeneratedValue(strategy = IDENTITY)
-    @Column(name = "recipe_id")
     private Long id;
     private String recipeTitle;
     private String recipeDescription;
-    @OneToOne(mappedBy = "recipe", cascade = ALL, fetch = LAZY, optional = false)
-    private CategoryModel categoryModel;
-    @OneToOne(mappedBy = "recipe", cascade = ALL, fetch = LAZY, optional = false)
-    private IngredientModel ingredientModel;
-    @OneToOne(mappedBy = "recipe", cascade = ALL, fetch = LAZY, optional = false)
-    private ProcedureModel procedureModel;
+    @ManyToOne(fetch = LAZY)
+    @JoinColumn(name = "categories_id",nullable = false)
```

Comment:
change column name from "categories_id" into "category_id"

Reply:
Already change! thank you 👍🏼